Source/WTF/wtf/Language.cpp

```diff
@@ -146,6 +136,16 @@ Vector<String> userPreferredLanguages(ShouldMinimizeLanguages shouldMinimizeLang
 
 #if !PLATFORM(COCOA)
```

Comment:
I know you didn't make this design originally, but I think we generally try to put non-Cocoa stuff in the SomethingGTK.cpp etc. files instead of in the non-platform-specific file and surrounding it with `!PLATFORM(COCOA)`.

Reply:
Here this is code shared by all ports except Cocoa so I think this is the right location for this function. I don't see any use in duplicating the logic for win and GTK.